verification/VerifyTasks.wdl

```diff
@@ -106,6 +106,28 @@ task CompareGtcs {
   }
 }
 
+task CompareTabix {
+  input {
+    File test_fragment_file
+    File truth_fragment_file
+  }
+  command {
+    a="md5sum ~{test_fragment_file}"
```

Comment:
Do we expect these to be identical? No header or anything?

Reply:
There's no header so the columns should be identical. Non-determinism may get us here, but it would be a problem for any comparison we do.